_integration-schemas/codat/v1/invoices.md

```diff
@@ -12,8 +12,8 @@ description: |
 replication-method: "Full Table"
```

Comment:
This rep method isn't correct. [Look at streams.py](https://github.com/singer-io/tap-codat/blob/master/tap_codat/streams.py#L458).